# Prevent system OpenBLAS from being used instead of the toolchain one
# Author: Davide Vanzo (Vanderbilt University)
diff -ru EIG-8.0.0.orig/src/Makefile EIG-8.0.0/src/Makefile
--- EIG-8.0.0.orig/src/Makefile	2019-08-07 16:21:10.518219095 -0500
+++ EIG-8.0.0/src/Makefile	2019-08-15 10:46:26.809998948 -0500
@@ -1,6 +1,6 @@
-override CFLAGS += -I../include -I/usr/include/openblas
+#override CFLAGS += -I../include -I/usr/include/openblas
 #LDLIBS += -lgsl -lopenblas -lrt -lm
-override LDLIBS += -lgsl -lopenblas -lm -lpthread
+#override LDLIBS += -lgsl -lopenblas -lm -lpthread
 # Some Linux distributions require separate lapacke library
 # override LDLIBS += -llapacke
 # Mac additions using homebrew installations
@@ -9,8 +9,6 @@
 # Harvard Medical School O2 cluster additions
-ifdef SLURM_CONF
-override CFLAGS += -I/n/app/openblas/0.2.19/include -I/n/app/gsl/2.3/include
-override LDFLAGS += -L/n/app/openblas/0.2.19/lib -L/n/app/gsl/2.3/lib/
-endif
+CFLAGS := ${CFLAGS} -I../include
+LDLIBS := -lgsl -lopenblas -lm -lpthread
 
 ifeq ($(OPTIMIZE), 1)
 	CFLAGS += -O2
